packages/qwik/src/core/state/common.ts

```diff
@@ -69,7 +70,25 @@ const _verifySerializable = <T>(value: T, seen: Set<any>): T => {
       case 'number':
         return value;
     }
-    throw qError(QError_verifySerializable, unwrapped);
+    let message = '';
+    if (preMessage) {
+      message = preMessage;
+    } else {
+      message = 'Value cannot be serialized';
+    }
+    if (ctx !== '_') {
+      message += ` in ${ctx},`;
+    }
+    if (typeObj === 'object') {
+      message += ` because it's an instance of "${value?.constructor.name}". You might need to use 'noSerialize()' or use a object literal instead. Check out https://qwik.builder.io/docs/advanced/dollar/`;
```

Comment:
```suggestion
      message += ` because it's an instance of "${value?.constructor.name}". You might need to use 'noSerialize()' or use an object literal instead. Check out https://qwik.builder.io/docs/advanced/dollar/`;
```